src/renderer/components/TLayouts/TScrollbar.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-	<vue-scrollbar :style="{ height: height, width: width }">
+	<vue-scrollbar
+		:style="{ height: height, width: width, margin: 0, padding: padding }"
```

Comment:
What is the idea behind using padding as props?

Reply:
I was doing a test and I forgot it. I just removed the prop now!